src/Starlight/System.hs

```diff
@@ -86,16 +77,18 @@ neighbourhoodOf c sys@System{ bodies, characters } = sys
   -- FIXME: laser power, not radar power, determines laser range
   -- FIXME: radar reflections
   -- FIXME: sharing radar with allies
+  -- FIXME: dimensional analysis
   visible i a = case i of
     B (Star _) -> True
     _          -> received > threshold
     where
     r = qd (a^.actor_.position_) (c^.actor_.position_)
     received = Watts ((c^.ship_.radar_.power_.unitary * gain * aperture * prj (a^.actor_.magnitude_) * patternPropagationFactor ** 4) / ((4 * pi) ** 2 * prj (r ** 2)))
   patternPropagationFactor = 1
-  gain = 1000000
-  aperture = 1000000
-  threshold = Watts 1.0e-12 -- 1 picowatt
+  gain = 1
+  aperture = 1000
+  threshold :: Watts Double
+  threshold = 1.0e-12
```

Comment:
One consequence of this not being particularly unit-safe and increasing from km to Mm is that we get to fudge by much smaller factors since the calculations are happening with 1000x smaller magnitudes already.